application/src/main/java/org/togetherjava/tjbot/logging/discord/DiscordLogForwarder.java

```diff
@@ -199,12 +198,17 @@ private static String describeLogEvent(LogEvent event) {
             return logMessage + "\n" + exceptionWriter.toString().replace("\t", "> ");
         }
 
-        private static @Nullable String linkToSource(String source, String sourceCodeBaseUrl) {
+        private static Optional<String> linkToSource(String source, String sourceCodeBaseUrl) {
             if (!source.startsWith(BASE_PACKAGE)) {
-                return null;
+                return Optional.empty();
+            }
+
+            if (!sourceCodeBaseUrl.endsWith("/")) {
```

Comment:
Can this not be done in Config? That way you don't need to always do this if-check each time `linkToSource` is called.

Reply:
not sure if this is a good idea though, maybe its intended to be without "/" and used with this consideration in other parts later, it would make more sense to change it on the DiscordLogForwarder level

Reply:
Sure thing